src/pixel/state.ts

```diff
@@ -157,19 +73,21 @@ export class Query {
 
 export class StateWrapper {
   data: State
-  eventBus: EventBus
+  eventBus?: EventBus
 
-  constructor (state: State, eventBus: EventBus) {
+  constructor (state: State, eventBus?: EventBus) {
```

Comment:
Don't we always provide a bus? Why does it need to be optional?

Reply:
Mostly missing in a lot of the tests. Same thinking as here: https://github.com/LiveIntent/live-connect/pull/103/files/bfcffb59ee3111e0f9d47c6e4b564b0b3ceed0ad#r1087556144

Let's decide whether we want these mandatory or not, will adjust accordingly